src/lib/dao/providers/ZdaoSdkProvider.tsx

```diff
@@ -27,19 +31,18 @@ type DaoSdkProviderProps = {
 export const ZdaoSdkProvider: React.FC<DaoSdkProviderProps> = ({
 	children,
 }) => {
-	const { library } = useWeb3React(); // get provider for connected wallet
-	const chainSelector = useChainSelector();
+	const { library, chainId } = useWeb3React<Web3Provider>(); // get provider for connected wallet
 
 	const [instance, setInstance] = useState<SDKInstance | undefined>();
 
 	const { network, selectedChain } = useMemo(() => {
-		const selectedChain = chainSelector.selectedChain;
+		const selectedChain = chainId || defaultNetworkId;
```

Comment:
Could be handy to have some library methods for parsing a chain ID:

```
If no chain ID, use default chain ID
else return chainIdToNetworkType
```

This will reduce some repitition